patches/chrome-browser-resources-signin-profile_picker-profile_creation_flow-local_profile_customization.html.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/resources/signin/profile_picker/profile_creation_flow/local_profile_customization.html b/chrome/browser/resources/signin/profile_picker/profile_creation_flow/local_profile_customization.html
+index 429b10a04532c4396002e74ff4693241a6a0cf85..4e34cad1382dd943957015ea499cd496bf22c330 100644
+--- a/chrome/browser/resources/signin/profile_picker/profile_creation_flow/local_profile_customization.html
++++ b/chrome/browser/resources/signin/profile_picker/profile_creation_flow/local_profile_customization.html
+@@ -152,6 +152,7 @@
+   #doneButton {
+     width : 32px;
+   }
++  .banner { visibility: hidden; }
```

Comment:
cc @petemill can these patches be avoided or made more extensible?

Reply:
well yes quite possibility but it would require introducing a new target for brave_overrides of the Signin page at `brave/browser/resources/signin` like we have done for the Settings page at `brave/browser/resources/settings`

Reply:
can we look at that as a follow-up?